### DISCUSSION TITLE
About skywalking Business Edition

### DISCUSSION THREAD

**Discussion by qigeqigeqi** [apache/skywalking#10850] [Q&A]
I would like to know which monitoring systems based on commercial versions of skywalking are available on the market.

**Comment by Superskyyy** [comment]
fyi @wu-sheng

**Comment by wu-sheng** [comment]
We don't know, and we don't care.
Apache 2.0 granted anyone and vendor to do commercial redisdistribution or service support 
We can't say for the world.
